src/viewport/index.tsx

```diff
@@ -333,6 +318,13 @@ class Viewport extends React.Component<ViewportProps & DefaultProps, ViewportSta
         if (prevProps.height !== height || prevProps.width !== width) {
             this.visGeometry.resize(width, height);
         }
+        if (prevState.showRenderParamsGUI !== this.state.showRenderParamsGUI) {
```

Comment:
this is where the show/hide happens.  this component has a piece of state called showRenderParamsGui but it has to be explicitly done through the visGeometry api below.